src/index.html

```diff
@@ -15,16 +15,16 @@
     <meta name="apple-mobile-web-app-status-bar-style" content="default">
     <meta name="msapplication-tap-highlight" content="no">
     <% if (htmlWebpackPlugin.options.manifest.theme_color) { %>
-      <meta name="theme-color" content="<%= htmlWebpackPlugin.options.manifest.theme_color %>">
+      <!-- <meta name="theme-color" content="<%= htmlWebpackPlugin.options.manifest.theme_color %>"> -->
```

Comment:
Was it intentionally commented and committed? Shouldn't you remove whole `if` block?